package.json

```diff
@@ -40,6 +40,8 @@
   "devDependencies": {
     "@types/react": "^17.0.19",
     "@types/react-dom": "^17.0.9",
-    "gh-pages": "^3.2.3"
+    "@types/styled-components": "^5.1.14",
+    "gh-pages": "^3.2.3",
+    "styled-components": "^5.3.1"
```

Comment:
it is more obvious to have "styled-components" in the `dependencies` section rather than in  `devDependencies`